### ISSUE TITLE
Filemanager

### ISSUE THREAD

**Issue by applibs** [zalify/easy-email#301]
Has anyone solved the implementation of a file manager for uploading images? Maybe Elfinder and others?

**Comment by leonardomelom** [comment]
In your editor, the uploader option doesn't show up?

**Comment by applibs** [comment]
> In your editor, the uploader option doesn't show up?

Yes it works. Read carefully what I wrote. File manager is not the same as uploaded.
